packages/@okta/vuepress-site/docs/concepts/user-profiles/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: User Profiles
+meta:
+  - name: description
+    content: A high-level, developer-focused overview of user profiles and the Okta Universal Directory.
+---
+
+# User Profiles
+
+A user profile in Okta is the data record where information about your user is stored. A typical user profile holds information, or attributes, like a user’s first name, last name, username, and email address. Users can be employees, customers, partners, or end-users of applications.
+
+The default Okta user profile has [31 user attributes](/docs/reference/api/users/#default-profile-properties), which can be customized based on client requirements. Additional custom attributes can be added to the user profile to support most client user needs. Custom profile attribute types further provide the ability to customize the user experience based on your org and application needs.
+
+You can manage user profile design and customization for your org, as well as individual user updates, from the Admin Console or using specific APIs.
+
```

Comment:
Remove hard return (line 15) so lines 14 and 16 is a paragraph as the cross-ref is relevant to line 14.

Reply:
The xref is relevant to the entire section; it's not specific to line 16.